### PR TITLE
Update Logging Cheat Sheet

### PR DESCRIPTION
**Pull request by sergiomarotco**
Added network architecture example in new chapter.
Logging_Cheat_Sheet drawio

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [x] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [x] All your assets are stored in the **assets** folder.
- [x] All the images used are in the **PNG** format.
- [x] Any references to websites have been formatted as [TEXT](URL)
- [x] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #<insert number here>.

Thank you again for your contribution :smiley:

### CODE REVIEW

File: cheatsheets/Logging_Cheat_Sheet.md [hunk 1]
```diff
@@ -236,6 +236,32 @@ Logging functionality and systems must be included in code review, application t
...
+    - It is allowed to combine the functionality of two applications in one
+
+For example, all external requests from users go through the API management service, see application in 'MIDDLEWARE 2' segment.
+
+![MIDDLEWARE](https://github.com/OWASP/CheatSheetSeries/blob/master/assets/Logging_Cheat_Sheet.drawio.png)
+
+As you can see in the image above, at the network level, the processes of saving and downloading logs require opening different network accesses (ports), arrows are highlighted in different colors. Also, saving and downloading are performed by different applications.
+
+Full network segmentation cheat sheet by [sergiomarotco](https://github.com/sergiomarotco): [link](https://github.com/sergiomarotco/Network-segmentation-cheat-sheet)
+
```
Comment (szh) [inline cheatsheets/Logging_Cheat_Sheet.md:264]: Perhaps we should wait to link to this cheat sheet until it's released in this repo (ie until #1036 is merged). I don't think we should link to a cheat sheet in a third-party repo.

**Review by szh — CHANGES_REQUESTED** [review]
Looks good overall. I left one comment, also please fix the lint errors.

File: cheatsheets/Logging_Cheat_Sheet.md (hunk 1 above)
Comment (sergiomarotco) [reply cheatsheets/Logging_Cheat_Sheet.md:264]: @szh I do not understand what character to use instead of quotes, just remove it?

Comment (szh) [reply cheatsheets/Logging_Cheat_Sheet.md:264]: I would leave this line out until the full cheat sheet is merged.

Comment (sergiomarotco) [reply cheatsheets/Logging_Cheat_Sheet.md:264]: @szh while we wait for another request to be approved, could you invite the community to discuss these edits

**Review by jmanico — APPROVED** [review]

**Comment by jmanico** [comment]
Forgive my late answer here. There are a few lint errors that block the merge, can you address those please?

**Comment by mackowski** [comment]
@jmanico you must revert these changes - this PR had a lint errors and now the side will not build, and all new PR will be failing.

**Comment by sergiomarotco** [comment]
@mackowski  see Por #1046

**Pull request merged**